### ISSUE TITLE
Feature request: set issue status after merging a MR

### ISSUE THREAD

**Issue by dj-hedgehog** [tf/redmine_merge_request_links#2]
The following is a quite specialized use-case but I want to propose it anyways.

Every time I accept a merge request in GitLab I change the affected issues status to `Testable`.
Would you find such a feature helpful for you too? It wouldn't need a UI for now as you could only change the issue status when an environment variable is set (pretty much like you did it with the AUTH token).

**Comment by tf** [comment]
Yeah, I thought about that as well. One could also consider integrating with or replicating Redmine's  functionality to [update issues via keywords in commit messages](https://github.com/redmine/redmine/blob/c04ab14ed4843baa0a7d55fe61a7377bb4909880/app/models/changeset.rb#L225) (e.g. `closes #123`). 

But even in the simpler case you describe, I'm not sure how to handle issues with  multiple associated merge requests. Does merging the first merge request trigger the action? Do all merge requests need to be merged?

Anyway, this feature is not a high priority for us at the moment. So for now I have no plans to work on it. If there is a well thought through proposal that answers the above questions, I'm happy to accept a PR.

I'll leave this issue open to collect further input on the idea.

**Comment by sandratatarevicova** [comment]
I have submitted a pull request which adds support for changing issue status after merging.

> Does merging the first merge request trigger the action?

We have only one merge request per issue in our workflow, so yes, merging the first merge request triggers the action. But some environment variable which would allow the second option could be added as well.
